soc/arm/nxp_imx/rt6xx/quick_access.ld

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2020 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+. = ALIGN(4);
+KEEP(*(CodeQuickAccess))
+KEEP(*(DataQuickAccess))
```

Comment:
Does the nxp hal have symbols in these sections?

Reply:
QuickAccess sections is used in SDK infrastructure when we need certain sections of code to be available in RAM instead of FLEXSPI.